### ISSUE TITLE
[de] detect missing spaces between word and brackets

### ISSUE THREAD

**Issue by GiantCrocodile** [languagetool-org/languagetool#873]
I'm not aware of a case where a word plus a bracket without a space in between is valid. Examples:

```
This should(not) be valid.
```

or

```
Dies führt(meistens) zu Problemen.
```

**Comment by f-knorr** [comment]
How about:
Verkäufer(in) gesucht!

**Comment by GiantCrocodile** [comment]
In my point of view this is no real sentence and not used in texts, only in ads/job seeking offers. I doubt that anybody would write it like this. Furthermore I think the real correct form would be:

 Verkäufer/-in gesucht!

**Comment by Discostu36** [comment]
In German this is allowed, also for something like "Gewinn(anteil)". See https://www.duden.de/sprachwissen/rechtschreibregeln/klammern

**Comment by GiantCrocodile** [comment]
True, `Gewinn(anteil)` is a good case. Maybe we can have some logic or basic check to see if just a space is missing or if it's valid wording. I'm not familiar about how much is possible in LanguageTool, thus I can't rate the possibility of my idea.

**Comment by Discostu36** [comment]
Maybe it'd be possible that LT checks if a correct word can be created by removing the bracket (e.g. `Gewinn(anteil)` --> `Gewinnanteil`). If this is not the case, a space could be suggested. This is beyond my skills but maybe somebody else could make a rule for that?

**Comment by marshinkar** [comment]
Both examples are now found by LT.

- This should(not) be valid.
- Dies führt(meistens) zu Problemen.

I am closing the issue.
